lib/kubeclient/common.rb

```diff
@@ -202,6 +205,14 @@ def resource_name(entity_type)
         entity_type.pluralize.downcase
       end
 
+      def proxy_url_service(namespace, service, protocol)
```

Comment:
@oschreib So you're mixing port & protocol into one? Port isn't required btw by the  API server proxy so should be made optional. Nor is protocol...